### DISCUSSION TITLE
Nested navigation group in V3

### DISCUSSION THREAD

**Discussion by ManojKiranA** [filamentphp/filament#8033] [Ideas]
In v2 it it totally possible to add multilevel menu after the [PR](https://github.com/filamentphp/filament/pull/4650) by @Nuhel 

But unfortunately the API that is implemented in v3 gives the error such as `Navigation group [Test Nav Group] has a nested group, which is not supported in the sidebar design at the moment.`

**Comment by vonec** [comment]
+100 for this feature  🙂

**Comment by ifly-leonard** [comment]
Yup! This feature is dearly missed.

**Comment by danharrin** [comment]
Someone come up with a good design then, we couldn't

**Reply by ManojKiranA** [reply]
What happened to design from v2. I think if we remove the icon starting from level 2 it looks fine right ?

**Reply by danharrin** [reply]
We decided it wasn't good enough to reintroduce into v3, but we welcome any PRs where we can discuss a proposed design. Be aware that the review will be harsh and probably quite strict

**Comment by Nuhel** [comment]
> Someone come up with a good design then, we couldn't

Will love to make an pr. 

What is the rules to follow for the design?

**Reply by danharrin** [reply]
@zepfietje do you have a set of requirements or do we just need to see what needs improving when the PR is submitted?

**Reply by zepfietje** [reply]
Hard to say upfront I think, because if we knew, we would probably have been able to come up with a design already. 😅
So yeah, go ahead and create the PR, then we'll test it for edge cases and such.

**Comment by Nuhel** [comment]
Hi, I have just stared working on this.

Any suggestion will be appreciated :)

![image](https://github.com/filamentphp/filament/assets/30118433/119ec6ea-d65f-45dd-8697-9e2a82e3c8cf)

**Reply by danharrin** [reply]
Personally I dont think the indentation looks right. If you make a PR, the review process is easier

**Reply by zepfietje** [reply]
Agree.

**Reply by dissto** [reply]
When in doubt copy WordPress' navigation design 😁
